src/frontend/src/components/Charge/index.tsx

```diff
@@ -30,14 +30,10 @@ export default class Charge extends React.Component<Props> {
       if (disposition.amended) {
         dispositionEvent += " (Amended)"
       }
-      return <>
+      return (
         <li className="mb2">
           <span className="fw7">Disposition:</span> {dispositionEvent}
-        </li>
-        <li className="mb2">
-          <span className="fw7">Charged: </span> {date}
-        </li>
-      </>
+        </li> )
     };
 
     const dispositionRender = (disposition: any, date: any) => {
```

Comment:
not related to this development, but any reason why known/null disposition is split into separate functions?  seems like they're logically the same thing, just with the null case split into its own function.  I would suggest splitting it into a function which makes the disposition string from the disposition object, then have the dispositionRender function create the actual html?

Reply:
yep good point, done.